chimney/src/test/scala/io/scalaland/chimney/DslSpec.scala

```diff
@@ -912,6 +912,38 @@ object DslSpec extends TestSuite {
         Bar1(1, Baz(Some(Bar1(2, Baz(None))))).transformInto[Bar2] ==> Bar2(Baz(Some(Bar2(Baz(None)))))
       }
     }
+
+    "support macro dependent transformers" - {
```

Comment:
Similar test for `TransformerF` would be much appreciated.

Reply:
Added test for TranformerF with safe option unwrapping rule. Declined #155, cause now such rule can be defined by implicits